src/types/CommonComponentProps.ts

```diff
@@ -0,0 +1,9 @@
+import {GridSize} from '@material-ui/core';
+
+interface CommonComponentProps {
```

Comment:
[nit] could we make this slightly more specifically named? something like 'CommonFieldComponentProps'? So we can know that it's a base type for the various form field components as opposed SQForm, SQFormDialog, or anything else. Maybe that's self-evident? Thinking out loud here...

Reply:
Maybe BaseFieldProps?